sdk/eidolon_ai_sdk/cpu/llm/mistral_llm_unit.py

```diff
@@ -134,9 +126,41 @@ class MistralGPTSpec(BaseModel):
 
 class MistralGPT(LLMUnit, Specable[MistralGPTSpec]):
     def __init__(self, **kwargs):
+        super().__init__(**kwargs)
         LLMUnit.__init__(self, **kwargs)
         Specable.__init__(self, **kwargs)
 
+    def get_models(self) -> List[LLMModel]:
```

Comment:
TODO for a followup pr: models should probably be a field on the spec with a default. I think it is important that models can be added dynamically, so we aren't gating people with what is basically config.

Reply:
I added it